server/create.go

```diff
@@ -467,7 +467,7 @@ func quantizeLayer(layer *layerGGML, quantizeType string, fn func(resp api.Progr
 		return nil, err
 	}
 
-	f, _, err := ggml.Decode(temp, 1024)
+	f, _, err := ggml.Decode(temp, -1)
```

Comment:
This shouldn't need to be updated since the output doesn't get fed back into `WriteGGUF`.